resources/aws/security_group.go

```diff
@@ -181,3 +256,29 @@ func (s SecurityGroup) GetID() (string, error) {
 
 	return *securityGroup.GroupId, nil
 }
+
+func (s SecurityGroup) getGroupFilters() ([]*ec2.Filter, error) {
```

Comment:
I think there isn't a reason to move that to a separate method, but I'm fine with that.